bibimainapp/pom.xml

```diff
@@ -20,48 +21,59 @@
             <url>https://jitpack.io</url>
         </repository>
     </repositories>
-    
-    
+
+
     <dependencies>
-        
+
         <!-- internal dependecies -->
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>bibimaininterfaces</artifactId>
-            <version>1.1</version>        
+            <version>1.1</version>
         </dependency>
-        
+
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>bibiservsearch</artifactId>
-            <version>1.1</version>        
+            <version>1.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-config</artifactId>
+            <version>3.0.1.RELEASE</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-jdbc</artifactId>
+            <version>3.0.1.RELEASE</version>
         </dependency>
-        
-        
+
         <!-- https://mvnrepository.com/artifact/com.sun.jersey/jersey-server -->
         <dependency>
-            <groupId>com.sun.jersey</groupId>
-            <artifactId>jersey-server</artifactId>
-            <version>1.2</version>
+            <groupId>org.glassfish.jersey.containers</groupId>
```

Comment:
Updating Jersey from 1.2 to 2.1 leads to a not working REST API for deployed application during Runtime. However the build in REST API (/manager/**) seem to be working fine with the new release.